### PR TITLE
Replaces Button by ButtonPlain component

### PR DESCRIPTION
**Pull request by jeffersontuc**
#### What problem is this solving?

Since now we have a more appropriate component to use, this PR replaces `Button` component by the `ButtonPlain` one.

#### How should this be manually tested?

[Workspace](https://buttonplain--checkoutio.myvtex.com/cart)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] [Linked this PR to a Clubhouse story (if applicable).](https://app.clubhouse.io/vtex/story/25532/substituir-button-de-cupom-por-buttonplain)
- [x] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/Coupon.tsx [hunk 1]
```diff
@@ -78,21 +78,15 @@ const Coupon: StorefrontFunctionComponent = () => {
...
           {!coupon && (
-            <div className="mb5">
-              <Button
-                id="add-coupon"
-                variation="tertiary"
-                collapseLeft
-                noUpperCase
-                onClick={toggle}
-              >
+            <div>
```
Comment (omninando) [inline react/Coupon.tsx]: Do you really need this `<div>`?

Comment (omninando) [inline react/Coupon.tsx]: Do you really need this `<div>`?

Comment (jeffersontuc) [reply react/Coupon.tsx]: For some reason, we do. Without the `<div>` the component uses the entire summary width.  See below:

image

Comment (jeffersontuc) [reply react/Coupon.tsx]: That happens when the new version of `checkout-summary` (which its gonna be up soon), that uses `flex-layout` on coupon's  block, is linked.

**Pull request merged**